app/src/main/java/org/luxoft/sdl_core/AndroidSettings.java

```diff
@@ -0,0 +1,89 @@
+package org.luxoft.sdl_core;
+
+import android.util.Log;
+
+import java.util.HashMap;
+
+public class AndroidSettings {
+
+    private static final String AndroidSection = "ANDROID";
+    private static final String TAG = AndroidSettings.class.getSimpleName();
+
+    private static HashMap<String, String> mValues;
+
+    public enum IniParams {
+        BUFFER_SIZE("BufferSize"),
+        READER_SOCKET_ADDRESS("ReaderSocketAdress"),
+        CONTROL_SOCKET_ADRESS("ControlSocketAdress"),
+        WRITER_SOCKET_ADRESS("WriterSocketAdress"),
+        PREFFER_MTU("PrefferredMtu"),
+        SDL_TESTER_SERVICE_UUID("SdlTesterServiceUUID"),
+        MOBILE_NOTIFICATION_CHARACTERISTIC("MobileNotificationCharacteristic"),
+        MOBILE_RESPONCE_CHARACTERISTIC("MobileResponceCharacteristic");
+
+        public final String param;
+
+        IniParams(String param) {
+            this.param = param;
+        }
```

Comment:
@VadymLuchko what is it for? Is this really needed? Maybe you can use standard `toString()`?